php/pro/Client.php

```diff
@@ -196,12 +196,13 @@ public function connect($backoff_delay = 0) {
     }
 
     public function send($data) {
-        $message = is_string($data) ? $data : Exchange::json($data);
-        if ($this->verbose) {
-            echo date('c'), ' sending ', $message, "\n";
-        }
-        $this->connection->send($message);
-        return null;
+        return React\Async\async(function () use ($data) {
```

Comment:
@pcriadoperez we have some other references to client.send, since it is now async should we update them as well?
![image](https://user-images.githubusercontent.com/43336371/232819099-37895319-8947-4072-8562-4656d281b185.png)